ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesSystemInfo/SeriesSystemInfo.js

```diff
@@ -16,6 +17,18 @@ export class SeriesSystemInfo extends Component {
         value: toShortDateTime(series.updated),
       },
     ];
+    if (!isEmpty(series.metadata.created_by)) {
```

Comment:
Very minor: shall we always display the field but with empty value, just to avoid to hide/show fields? (and support tickets in the future asking why the field is not displayed :))

pseudocode:
```
      const created_by_type = _get('series.metadata.created_by.type')
      const created_by_value = _get('series.metadata.created_by.value', '-')
      const created_by = created_by_type ? `{created_by_type}: {created_by_value}` : {created_by_value}
      rows.push({
        name: 'Created by',
        value: `${created_by}`,
      });
```